Source/control.cpp

```diff
@@ -2366,63 +2366,41 @@ void DrawDurIcon()
 // 4B8968: using guessed type int sbookflag;
 // 69BD04: using guessed type int questlog;
 
-int DrawDurIcon4Item(ItemStruct *pItem, int x, int c)
+int DrawDurIcon4Item(ItemStruct *item, int x, int frame)

```

Comment:
When updating function signatures, also remember to update the names of parameters in the C header file.

From `Source/control.h`:
```c
int DrawDurIcon4Item(ItemStruct *pItem, int x, int c);
```